src/qt/bitcoin.cpp

```diff
@@ -563,6 +558,11 @@ int main(int argc, char *argv[])
 
     std::unique_ptr<interfaces::Node> node = interfaces::MakeNode();
 
+    // Subscribe to global signals from core
+    std::unique_ptr<interfaces::Handler> handler_message_box = node->handleMessageBox(noui_ThreadSafeMessageBox);
+    std::unique_ptr<interfaces::Handler> handler_question = node->handleQuestion(noui_ThreadSafeQuestion);
+    std::unique_ptr<interfaces::Handler> handler_init_message = node->handleInitMessage(noui_InitMessage);
+
     // Do not refer to data directory yet, this can be overridden by Intro::pickDataDirectory
```

Comment:
What makes it safe to subscribe to them here already, when some of then log to the debug log, which is in the datadir?

Reply:
If that was the case then `boost::signals2::no_slots_error` would have been raised before?